Assets/__Scripts/MapEditor/UI/KeybindUpdateUIController.cs

```diff
@@ -52,10 +55,26 @@ public void OnPlaceRedNoteorEvent(InputAction.CallbackContext context)
     {
         if (!context.performed) return;
         redToggle.onValueChanged.Invoke(true);
+        redSelected = true;
         placeMode.SetMode(PlacementModeController.PlacementMode.NOTE);
         lightMode.UpdateValue();
     }
 
+    public void OnToggleNoteorEvent(InputAction.CallbackContext context)
+    {
+        if (!context.performed) return;
+        if (redSelected)
```

Comment:
This should probably be checking the current `isOn` value of `blueToggle` or `redToggle` rather than trying to duplicate that state in another variable.

Reply:
Or, having looked at it, we seem to not currently be setting the toggles themselves. So either we can do that in `ColorTypeController` or check `leftSelected.enabled` which is being set to change the visual style.

Reply:
Checked `leftSelected.enabled`.